backend/lumens.ts

```diff
@@ -13,10 +14,22 @@ interface CachedData {
   };
 }
 
-export let cachedData: CachedData;
-
-export const v1Handler = function (_: any, res: Response) {
-  res.send(cachedData);
+export const v1Handler = async function (
+  _: any,
+  res: Response,
+  next: NextFunction,
+) {
+  try {
+    let cachedData = await redisClient.get("lumensV1");
+    if (cachedData == null) {
+      return next(Error("null value found"));
+    }
```

Comment:
I see that for a lot of these handlers you're following a pattern of checking `cachedData` and returning an error if it's null. Would it be helpful to create a helper that handles this so you don't have to keep doing this check manually?

Reply:
sgtm, I went back and forth originally but leaned towards [writing code that's easy to delete over easy to extend](https://github.com/stellar/dashboard/pull/205/commits/62b6f5890035aef35e5d8380804114b532a962b3) (an article I got from @leighmcculloch  😄 ). Looking at it again it does look cleaner with a helper though, added 👍 
https://github.com/stellar/dashboard/pull/205/commits/62b6f5890035aef35e5d8380804114b532a962b3